config/v1/feature_gates.go

```diff
@@ -0,0 +1,174 @@
+package v1
+
+type FeatureGateDescription struct {
+	// FeatureGateAttributes is the information that appears in the API
+	FeatureGateAttributes FeatureGateAttributes
+
+	// OwningJiraComponent is the jira component that owns most of the impl and first assignment for the bug.
+	// This is the team that owns the feature long term.
+	OwningJiraComponent string
```

Comment:
You have identified this as Jira component, a lot of the entries do not actually map to a jira component, what do you want to do here? Do we remap everything to a jira project? Or is this bug component?

Reply:
> You have identified this as Jira component, a lot of the entries do not actually map to a jira component, what do you want to do here? Do we remap everything to a jira project? Or is this bug component?



I think we'll fix it up as a followup.  We have other reasons to collect a list of all jira components

Reply:
Conversation out of band, this will eventually contain information to allow you to route a bug should there be an issue with the feature/tests related to the feature.